src/engine/QueryExecutionTree.cpp

```diff
@@ -97,6 +89,10 @@ QueryExecutionTree::selectedVariablesToColumnIndices(
     const SelectedVarsOrAsterisk& selectedVarsOrAsterisk,
     const ResultTable& resultTable) const {
   ColumnIndicesAndTypes exportColumns;
+
+  // Should we merge this if-else-clause into less code ?
```

Comment:
@joka921 What' your opinion about it ?

Reply:
My suggestion is the following:
When a variable from the query (`AccordinglySelected`) is not found, 
Always emit a warning (no matter if `isAsterisk` or not, so you also warn, 
When somebody selects a variable, which doesn't exist.
For the other direction I agree with you, we have tests for this and you can remove the warning.